programs/drift_vaults/src/state/vault.rs

```diff
@@ -102,24 +112,32 @@ impl Vault {
 }
 
 impl Size for Vault {
-    const SIZE: usize = 528 + 8;
+    const SIZE: usize = 504 + 8;
```

Comment:
shouldn't the above be [u64; 7] and this should still be 528, otherwise it's a breaking change?